### DISCUSSION TITLE
OSMT CLI Quickstart Help

### DISCUSSION THREAD

**Discussion by x360539** [wgu-opensource/osmt#464] [Asking for Help] [answered]
I am installing OSMT for the first time and attempting to follow the instructions in the OSMT How-To document. I am at the point "Starting a local OSMT", Step 4, where the instructions say to run "./osmt_cli.sh -q". However, for my osmt_cli.sh, "-q" is not an available argument. When I attempt to run  "./osmt_cli.sh -q", I receive the error message "ERROR: Invalid argument". In addition, when I run "./osmt_cli.sh -h", "-q" does not appear as an available argument. Thank you for any help you can provide.

**Comment by JohnKallies** [comment, marked as answer]
`osmt_cl.sh -q` supported a local OSMT stack called a "Quickstart". The original intention was a local demo instance. We found it to be unreliable and awkward, and the code as it exists on the develop branch no longer supports a Quickstart. By "How To" pages, I'm guessing you mean on the osmt.io site. Unfortunately, some of that content is becoming stale. The README.md file in the project root is the strongest source of truth.

Try following the guidance here:
* https://github.com/wgu-opensource/osmt/blob/develop/README.md#running-the-development-configuration

**Reply by x360539** [reply]
Thank you so much!
